evalml/tests/dependency_update_check/latest_dependency_versions.txt

```diff
@@ -1,6 +1,5 @@
 black==23.7.0
 catboost==1.2
-category-encoders==2.5.1.post0
```

Comment:
`category-encoders` is an actual dependency of ours, and it is pinned in `pyproject.toml`. Is this something we want to be removing from the file?

Reply:
Ah I assumed that it was superseded by scikit-learn, but looks like it's actually separate based on [this](https://github.com/alteryx/evalml/pull/1401/files). I'll add it back in!